packages/tokens/README.md

```diff
@@ -0,0 +1,3 @@
+# @fluentui/tokens
+
+**Fluent UI Theme Tokens**
```

Comment:
Would you mind to create an issue to fill this README in a separate PR?

Reply:
Yup, I will handle the readme in the next PR.